docs/installation.md

```diff
@@ -260,3 +261,38 @@ Hello, FISCO BCOS
 ```
 
 [build_chain_code]:https://github.com/FISCO-BCOS/FISCO-BCOS/blob/master/tools/build_chain.sh
+
+## 附录：Java环境配置
```

Comment:
move this to console or https://fisco-bcos-documentation.readthedocs.io/zh_CN/release-2.0/docs/faq.html

Reply:
moved